with open('10_FilesNExceptions\py_digit_2.txt') as file_object:
	contents = file_object.read()
	print(contents)
